src/wrapper.js

```diff
@@ -6,16 +6,19 @@ function accessibleAutocomplete (opts) {
   if (!opts.id) { throw new Error('id is not defined') }
   if (!opts.source) { throw new Error('source is not defined') }
   if (Array.isArray(opts.source)) {
-    opts.source = createSimpleEngine(opts.source)
+    opts.source = createSimpleEngine(opts.source, opts.showAllValues)
   }
   render(<Autocomplete {...opts} />, opts.element)
 }
 
-const createSimpleEngine = results => (query, syncResults) => {
-  const filteredResults = query
-    ? results.filter(r => r.toLowerCase().indexOf(query.toLowerCase()) !== -1)
-    : []
-  syncResults(filteredResults)
+const createSimpleEngine = (values, showAllValues) => (query, syncResults) => {
+  var matches = []
+
+  if (showAllValues || query !== '') {
```

Comment:
I'm not sure if we even need this (nor did we need it before)! It can just always return something, because the autocomplete won't call it unless it wants something.

Can we just delete the conditional?